src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -1432,6 +1432,25 @@ def _maybe_performance_problem(self) -> Optional[PerformanceProblem]:
             offender_span_ids=[span["span_id"] for span in offender_spans],
         )
 
+    def _total_span_duration(self, spans: Sequence[Span]) -> timedelta:
```

Comment:
Hey does this function help you here? 
https://github.com/getsentry/sentry/blob/d0cb533289557f085e19bc20a6f33f81a346fdbb/src/sentry/utils/performance_issues/performance_detection.py#L380

Reply:
damn already existed, I had a feeling it might have but couldn't find it, only found the relay one 🤔

Reply:
Hmm do we need to rename it for discoverability?

Reply:
I don't know if that would help I don't know why I didn't think to look in the file, I think I reviewed that PR too 😢

Reply:
Hooray!/Well crap. Thanks, will update to use the existing implementation 🤦

Reply:
Updated to use the existing method :+1:

Reply:
👍 existing is better